docs/modules/SDDs/nav.adoc

```diff
@@ -0,0 +1,22 @@
+.Syn Design Documents - SDD
```

Comment:
Should we maybe update the navigation per added SDD? We will most likely not add every SDDs at once which will eventually lead into broken navigation...

Reply:
That will lead to conflicts I guess, because all SDD branch out from `master` and would all modify the same empty file. I thought since we know what the filenames will be, just provide the navigation and it will eventually be fully working.

Reply:
OK, let's try. We can change it later anytime =)